### DISCUSSION TITLE
Share your Live Site!

### DISCUSSION THREAD

**Discussion by jlaksana** [hack4impact-calpoly/bootcamp-project-2023#156] [Show and tell]
Feel free to post your final live site here!

**Comment by SilveerDusk** [comment]
Here it is! 
[The Live Site](https://personal-website-pearl-three-29.vercel.app/)

**Comment by devinhadley** [comment]
[Site](https://bootcamp-project-2023-two.vercel.app/)

**Comment by elhagen13** [comment]
https://bootcamp-project-2023.vercel.app

**Comment by Robert303V** [comment]
[Here's my site!](https://robert303v.vercel.app/)

**Comment by soramicha** [comment]
My website: [sc-bootcamp-23.vercel.app](https://sc-bootcamp-23.vercel.app/)

**Comment by Belal-Elshenety** [comment]
Here it is!
[My Personal Website](https://belalelshenety.vercel.app/)

**Comment by xgarcia17** [comment]
Here is my website! https://xaviers-personal-website-git-main-xgarcia17s-projects.vercel.app/

**Comment by Westluu** [comment]
https://bootcamp-delta.vercel.app/

**Comment by aarav27** [comment]
Here is my site:
https://bootcamp-project-2023-six.vercel.app/

**Comment by Bdon101** [comment]
https://bdons-website.vercel.app/

**Comment by peterchinh** [comment]
Here's my website: https://bootcamp-project-2023-eight.vercel.app/

**Comment by tarannssingh** [comment]
https://personal-w-beta.vercel.app/

**Comment by mtbosio** [comment]
https://bootcamp-project-2023-matthew-bosio-2437qae5w-mtbosios-projects.vercel.app/

**Comment by ajesus7** [comment]
https://www.aidannesbitt.com/portfolio

**Comment by asarelcastellanos** [comment]
https://hack4impact-asarelcastellanos.vercel.app/

**Comment by jojayama** [comment]
https://bootcamp-project-2023-beryl.vercel.app/

**Comment by Sohini-pillay** [comment]
https://bootcamp-project-2023-6zwknv5jc-sohinis-projects.vercel.app/

**Comment by FlowerCake13** [comment]
[https://lindsay-minami-h4i-bootcamp-2023.vercel.app/](https://lindsay-minami-h4i-bootcamp-2023.vercel.app/)

**Comment by kaylantran** [comment]
https://mypersonalwebsite-9ufizno2u-k4yluhs-projects.vercel.app/

**Comment by lukewaltz** [comment]
https://luke-waltz-personal-website.vercel.app/

**Comment by sophiapeckner** [comment]
https://hack4impact-bootcamp.vercel.app/

**Comment by whiniy** [comment]
https://bootcamp-project-2023-smoky.vercel.app/

**Comment by MatthewBlam** [comment]
[Site](https://bootcamp-project-2024-sable.vercel.app/)
